src/server.tsx

```diff
@@ -189,11 +189,13 @@ app.post(
         const notTheSame = got?.test.name !== expectedTest || got?.variant.name !== expectedVariant;
 
         if (notBothFalsy && notTheSame) {
-            console.log({
-                status: 'comparison failed',
-                targeting,
-                frontendLog,
-            });
+            console.log(
+                JSON.stringify({
```

Comment:
We do the same thing (`console.log` + `JSON.stringify`) in the request logging middleware. I wonder if this justifies a `logJson` function or something 🤔

Reply:
yeah possibly - definitely an easy error to make!